buildkite/test-splitter
# Add option to set output file

### Description
in #156, we changed how we trigger the retry by saving the output of Rspec run to detect failed tests. The output of Rspec run was saved in a temporary file that will be deleted at the end of operation.

This PR adds a new option called `BUILDKITE_SPLITTER_RESULT_PATH` that can be used to specify the path to save the output of Rspec run. If the configured path has `*`, it will be replaced with a UNIX timestamp, this is useful when the retry function is enabled, and we want to store the output for each run separately without overriding the previous run.

### Testing
I've done integration testing manually
- set `BUILDKITE_SPLITTER_RESULT_PATH=./tmp/my-rspec-result-*.json`
- run `./test-splitter`
- check if a json file exists in `$BUILDKITE_SPLITTER_RESULT_PATH` configured above
